pkg/retriever/bitswapretriever.go

```diff
@@ -161,7 +161,7 @@ func (br *bitswapRetrieval) RetrieveFromAsyncCandidates(ayncCandidates types.Inb
 		return nil, nil
 	}
 
-	br.events(events.Started(br.clock.Now(), br.request.RetrievalID, phaseStartTime, types.RetrievalPhase, bitswapCandidate, multicodec.TransportBitswap))
+	br.events(events.StartedRetrieval(br.clock.Now(), br.request.RetrievalID, bitswapCandidate, multicodec.TransportBitswap))
```

Comment:
Let's have a think through the timing here and whether it makes sense:

* `startTime` is at the top of the function
* `asyncCandidates.Next()` may block, collecting the initial batch of candidates, spending some time
* `StartedRetrieval` gets the time just after collecting candidates
* `FirstByte` has a duration that's taken from `startTime`, which is prior to collecting candidates

In both Graphsync and HTTP, we do `FirstByte` from the start of the time we begin a retrieval-proper from that particular candidate, after candidate collecting and even after `Connect()`, it's from "can I haz plz".

I think, given that, it might make sense for `FirstByte` here to be from something closer to the `StartedRetrieval` time. But I'm not really sure, since one could argue that Bitswap is different enough that `FirstByte` has a different meaning.

Reply:
Issue #333 will track this